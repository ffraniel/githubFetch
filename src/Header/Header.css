.header {
    padding-bottom:15px;

    background-color:#373b41;
}
@media only screen and (max-width: 999px) {
    .header{
        padding-left:5%;
    }
}
@media only screen and (min-width: 1000px) {
    .header{
        padding-left:10%;
    }
}

#logo {
    width:33px;
    height:33px;
    color:white;
    filter: invert(100%);
    padding:10px;
    padding-top:10px;
    text-justify: center;

}