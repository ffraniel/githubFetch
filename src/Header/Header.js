import React, { Component } from 'react';
import './Header.css';
import Search from '../Search/Search';

class Header extends Component {
    render(){
        return(
        <header className="header">
          <h1 className="header-title">The Repo Room</h1>
          <Search handleChange={this.props.handleChange} handleSubmit={this.props.handleSubmit}/>
        </header>
        )
    }
}

export default Header;