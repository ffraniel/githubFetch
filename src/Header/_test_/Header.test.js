import React from 'react';
import ReactDOM from 'react-dom';
import { shallow, mount, render, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';
import Header from '../Header';

configure({ adapter: new Adapter() });

test('Header renders with title ', () => {
    const wrapper = mount(
        <Header />
    );
    const headerTitle = wrapper.find('.header-title');
    
    expect(headerTitle.text()).toBe("The Repo Room");

    const rendered = renderer.create(
        <Header />
    );
    expect(rendered.toJSON()).toMatchSnapshot();
})