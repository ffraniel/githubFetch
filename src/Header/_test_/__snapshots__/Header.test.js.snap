// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders with title  1`] = `
<header
  className="header"
>
  <h1
    className="header-title"
  >
    The Repo Room
  </h1>
  <form
    className="searchForm"
    onSubmit={undefined}
  >
    <input
      onChange={undefined}
      type="text"
      value={undefined}
    />
    <input
      type="submit"
    />
  </form>
</header>
`;
